### ISSUE TITLE
Mac ?

### ISSUE THREAD

**Issue by gedw99** [dirslashls/windows11-systray-gioui#1]
I have Mac and windows and can try on Mac if you want.

Using the fyne import with gio was something I was planning to try

**Comment by gedw99** [comment]
it works on MAC btw.

**Comment by dirslashls** [comment]
Yes, it works great on Mac. Issue seems to be only for Windows.

**Comment by gedw99** [comment]
I will give it a try on windows 11 Arm64 and push a PR if I get lucky

**Comment by dirslashls** [comment]
I experimented some more and got it working by using Run but with go. I updated the project with that option. Pleaes check. I hope this is a valid way to use systray with external UI.

**Comment by gedw99** [comment]
ok I pulled latest
on Mac when I press "Test" is does nothing...

**Comment by dirslashls** [comment]
Yes, the callback is not implemented but my issue was that the menu itself was not showing up. I will push a commit that launches a simple window.

**Comment by gedw99** [comment]
On ym end the menu shows up, I click, and no dialogue happens, and the menu goes dim

LaunchDialog is being called but not becoming visible.

**Comment by gedw99** [comment]
just realised that this will allow gio to have menus for non sys tray apps too.

**Comment by dirslashls** [comment]
It may be possible that the launched window is below your other windows. I provided a patch to gioui so that it always launches on top. It should be in the next version (post 0.3.1) when it is released.

**Comment by gedw99** [comment]
Will give it another try later. Dinner time here :)

**Comment by dirslashls** [comment]
That is surprising, --go works for me. Although, if I run it a couple of times, intermittently it won't open. When it does, it works any number of times within the same invocation of the program. Attached is a video of the program working.

[windows-gioui-systray.zip](https://github.com/dirslashls/windows11-systray-gioui/files/13493491/windows-gioui-systray.zip)

**Comment by gedw99** [comment]
Thanks for zip but I am on windows arm.

I guess I have to try and debug it on my Mac to see what’s not working.

I will add a makefile do we can both be sure we are running the same commands. Will pr it in.

**Comment by gedw99** [comment]
There is another gio project that has menus working for mac. 

We plan to add windows too.

we should try it out .

you got the link I think ? Sent it before I believe ..

**Comment by dirslashls** [comment]
I tried the 2fa app on Windows. The UI comes up but there is no icon in the tray.

**Comment by gedw99** [comment]
interesting ..

Could you make an issue on that repo so we can all work it out together.

GitHub actions for Mac and windows are enabled in that repo too so it will make it easier to debug what’s happening.

**Comment by gedw99** [comment]
You can enable windows builds in the makefile btw . If you need me to do it just shout
